acacia/env_to_config_override.py

```diff
@@ -0,0 +1,17 @@
+import os
+from ConfigParser import ConfigParser
+
+if __name__ == '__main__':
+
+    with open('/app/config', 'a') as ofile:
+        ofile.write('\n[override]\n')
```

Comment:
this section header is part of the checked in config file, should it be both places?